index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
+    <title>Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav-menu">
+            <div class="logo">
+                <a href="#" class="logo-text">My Portfolio</a>
+            </div>
+            <div class="hamburger-menu">
+                <a href="#">
+                    <i class="fa-solid fa-bars"></i>
+                </a>
+            </div>
+        </nav>
+    </header>
+    <main class="main">
+        <section class="headline">
+            <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
+            <p class="description-text"> I can help you build a product , feature or website
+                Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <div>
+                <p class="social-icons-header">Let's Connect</p>
+                <ul class="social-icons">
+                    <li><a href="https://twitter.com/2belamit" aria-label="Twitter" target="_blank" rel="noopener"><i
+                                class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin"
+                            target="_blank" rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href="https://github.com/belaymit?tab=repositories" aria-label="Github" target="_blank" rel="noopener"><i
+                                class="fa fa-github"></i></a></li>
+                    <li><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i
+                                class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </section>

```

Comment:
[Required] As requested by the previous reviewer please note that as per the Figma template provided this section needs to follow the design of the Figma template. Please add a background image to this section as shown on the design, in your current implementation of the design the background image is missing. Kindly rectify this.

## Figma template
<img src="https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/headline-mobile.png" width="300px" />

## Current Implementation
<img src="https://user-images.githubusercontent.com/16392046/198149500-4f97e168-7312-4a2f-be4d-387bf897cef5.png" width="300px" />